koku/api/organizations/queries.py

```diff
@@ -235,6 +235,10 @@ def _get_accounts_list(self, org_id):
                 account_query = account_query.exclude(created_timestamp__gte=self.end_datetime)
                 if self.query_filter:
                     account_query = account_query.filter(self.query_filter)
+                if self.access:
+                    accounts_to_filter = self.access.get("aws.account", {}).get("read", [])
+                    if accounts_to_filter and "*" not in accounts_to_filter:
+                        account_query = account_query.filter(account_alias__account_id__in=accounts_to_filter)
```

Comment:
Wouldn't the AWS account filtering happen here https://github.com/project-koku/koku/blob/c92af6b7324522229cfe0bf895b63b193f1f85bc/koku/api/organizations/aws/view.py#L33?